samcli/lib/utils/progressbar.py

```diff
@@ -22,4 +22,4 @@ def progressbar(length, label):
         Progressbar
 
     """
-    return click.progressbar(length=length, label=label, show_pos=True)
+    return click.progressbar(length=length, label=label, show_pos=True, file=stream)
```

Comment:
the progress bar was printed to stdout, although  it should printed to stderr as stdout is only for the lambda invoke output